src/Aspire.Hosting.Milvus/api/Aspire.Hosting.Milvus.cs

```diff
@@ -17,8 +17,11 @@ public static partial class MilvusBuilderExtensions
         public static ApplicationModel.IResourceBuilder<T> WithAttu<T>(this ApplicationModel.IResourceBuilder<T> builder, System.Action<ApplicationModel.IResourceBuilder<Milvus.AttuResource>>? configureContainer = null, string? containerName = null)
             where T : Milvus.MilvusServerResource { throw null; }
 
+        [System.Obsolete("Use WithConfigurationFile instead.")]
```

Comment:
Why obsolete it? Is there no scenario for this? @danegsta

Reply:
Only real reason to use a bind mount instead of copy is if you need to actively sync changes between the host and container, which doesn’t generally apply to initialization data/scripts. For scenarios like this where a read only bind mount was used previously, the new API provides the same functionality, but with better file ownership and permission behavior in the container.

Reply:
Do we have a deployment plan for container files? If not, we shouldn't obsolete these APIs.

Reply:
I think introducing container files in the core and using them for emulators is a good step, but lets stage these obsoletions with a better migration plan that we have time to noodle on.